projects/steering/src/steering_can.c

```diff
@@ -29,9 +32,19 @@ StatusCode steering_can_process_event(Event *e) {
       break;
     case STEERING_CONTROL_STALK_EVENT_LEFT_SIGNAL:
       CAN_TRANSMIT_LIGHTS(EE_LIGHT_TYPE_SIGNAL_LEFT, (EELightState)e->data);
+      if ((EELightState)e->data == EE_LIGHT_STATE_ON) {
+        turning = true;
+      } else {
+        turning = false;
+      }

```

Comment:
Boolean zen tip: you can just do `turning = ((EELightState)e->data = EE_LIGHT_STATE_ON)`.

Reply:
Oh yeah I thought about doing a turnery operator, but I think I was told against that somewhere.... I can change it